### DISCUSSION TITLE
JB2A/Sequencer integration

### DISCUSSION THREAD

**Discussion by MrZwij** [DFreds/dfreds-convenient-effects#74] [Ideas]
Hi! Have you given any thought to integrating with Sequencer/JB2A for visual effects? The TokenMagicFX effects are nice, but what can be done with those two modules is SUPER nice. 😄

**Comment by DFreds** [comment]
I haven't really considered it. Have you tried to do it yourself with any success? CE are just pre-defined active effects, so if they aren't possible with active effects, it probably isn't feasible.

**Reply by MrZwij** [reply]
I have not tried it myself, and I wouldn't know where to begin. I do know that Sequencer supplies an API to apply/remove visual effects from JB2A to the token. But I'm not technical enough to outline how to do it.

**Comment by DFreds** [comment]
I don't think this is possible without being able to attach a macro to an active effect itself. That might be a good idea for a module though 🤔
